pkg/sanity/controller.go

```diff
@@ -92,7 +92,7 @@ func isControllerCapabilitySupported(
 	return false
 }
 
-var _ = DescribeSanity("Controller Service", func(sc *SanityContext) {
+var _ = DescribeSanity("Controller Service [Controller Server]", func(sc *SanityContext) {
```

Comment:
That looks rather odd. Can't you simply skip based on the existing "Controller Service" string?

Reply:
yes, agreed it's redundant, but I think it's neater if everything has the same tag.

Reply:
Ah, I didn't know that this tag is already in use elsewhere. Okay, then it makes sense.